src/__tests__/to-have-style.js

```diff
@@ -11,10 +11,10 @@ describe('.toHaveStyle', () => {
         style={[
           {
             backgroundColor: 'blue',
-            height: '100%',
             transform: [{ scale: 2 }, { rotate: '45deg' }],
```

Comment:
Pls add `height: 40%' next to `backgroundColor: blue` & `transform` in order to test style overridding.

Reply:
Done, also added some more cases that were helpful while removing `mergeAll`